portality/models/v2/shared_structs.py

```diff
@@ -237,4 +240,48 @@
             }
         }
     }
+}
+
+ARTICLE_STRUCT = {
+    "fields" : {
+        "created_date": {"coerce": "utcdatetime"},
+        "es_type": {"coerce": "unicode"},
+        "id": {"coerce": "unicode"},
+        "last_updated": {"coerce": "utcdatetime"},
```

Comment:
ARTICLE_STRUCT is not a new addition. Only fields were added. I suggest to keep it separate so that article.py file will not be huge and keeps it tidy